src/y_doc.rs

```diff
@@ -415,6 +418,35 @@ pub fn apply_update(doc: &mut YDoc, diff: Vec<u8>) -> PyResult<()> {
     Ok(())
 }
 
+#[pyfunction]
+pub fn update_to_nodes(diff: Vec<u8>) -> PyObject {
+    let doc = Doc::new();
```

Comment:
Keep this function but replace with your implementation

Reply:
Done